pkgs/development/web/cypress/default.nix

```
@@ -73,7 +80,7 @@ stdenv.mkDerivation rec {
     mainProgram = "Cypress";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.mit;
-    platforms = [ "x86_64-linux" ];
+    platforms = [ "x86_64-linux" "aarch64-linux" ];
```

```suggestion
    platforms = lib.attrNames allBinaries;
```

Maybe the error was already kind of ok because of this line, but if so, this deduplication is still nice.